server/api/helper/tenant_helper.go

```diff
@@ -2,27 +2,34 @@ package helper
 
 import (
 	"fmt"
-	"github.com/gofrs/uuid"
 	"github.com/labstack/echo/v4"
 	"github.com/teamhanko/passkey-server/persistence/models"
-	"github.com/teamhanko/passkey-server/persistence/persisters"
 	"net/http"
+	"strings"
 )
 
-func FindTenantByIdString(id string, tenantPersister persisters.TenantPersister) (*models.Tenant, error) {
-	tenantId, err := uuid.FromString(id)
-	if err != nil {
-		return nil, echo.NewHTTPError(http.StatusBadRequest, "unable to parse tenant id").SetInternal(err)
+func CreateApiKeyError(keys []models.Secret, apiKey string) error {
```

Comment:
```suggestion
func CheckApiKey(keys []models.Secret, apiKey string) error {
```
I think the function name should describe wha the function does, with the current name I would think that always an error is returned.